numpy/core/src/multiarray/item_selection.c

```diff
@@ -1093,87 +1093,20 @@ _new_argsortlike(PyArrayObject *op, int axis, PyArray_ArgSortFunc *argsort,
 }
 
 
-/* Be sure to save this global_compare when necessary */
-static PyArrayObject *global_obj;
-
-static int
-sortCompare (const void *a, const void *b)
-{
-    return PyArray_DESCR(global_obj)->f->compare(a,b,global_obj);
-}
-
-/*
- * Consumes reference to ap (op gets it) op contains a version of
- * the array with axes swapped if local variable axis is not the
- * last dimension.  Origin must be defined locally.
- */
-#define SWAPAXES(op, ap) { \
```

Comment:
I confess I don't know what all these `SWAP*` macros were used for, but it is nice to see them gone.